### PR TITLE
PIP-1135-accession-dnase-analysis

### PR DESCRIPTION
**Pull request by paul-sud**

### CODE REVIEW

File: accession_steps/dnase_steps.json [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+          ],
+          "file_format": "bam",
+          "filekey": "analysis.nuclear_bam",
+          "output_type": "alignments",
+          "quality_metrics": ["nuclear_flagstats",
+                              "nuclear_duplication_metric",
+                              "nuclear_hotspot1_metric",
+                              "nuclear_samtools_stats",
+                              "nuclear_alignment_quality_metric"
+                            ]
```
Comment (paul-sud) [inline accession_steps/dnase_steps.json]: Formatting here is wonky and causing tests to fail. please run `tox -e lint` and push

**Review by paul-sud — COMMENTED** [review]
Please lint, looks good otherwise.

File: accession_steps/dnase_steps.json (hunk 1 above)
Comment (ottojolanki) [reply accession_steps/dnase_steps.json]: yup, also had some inferred type annotations that needed fixing.

Comment (ottojolanki) [reply accession_steps/dnase_steps.json]: I am not familiar with mypy, would you mind checking the typing mypy complaints were resolved in the "correct" way (it is not complaining, but not sure if this is the way it is supposed to be done). This https://github.com/ENCODE-DCC/accession/compare/PIP-1135-accession-dnase-analysis#diff-959e6b6869da1fe061fc47c5620f48a4R1242 and this https://github.com/ENCODE-DCC/accession/compare/PIP-1135-accession-dnase-analysis#diff-959e6b6869da1fe061fc47c5620f48a4R1298. I am ready to merge if you are satisfied with how those are.

Comment (paul-sud) [reply accession_steps/dnase_steps.json]: hints look good, thanks for adding

**Pull request merged**